book/10-git-internals/sections/objects.asc

```diff
@@ -242,6 +242,8 @@ $ echo 'first commit' | git commit-tree d8329f
 fdf4fc3344e67ab068f836878b6c4951e3b15f3d
 ----
 
+You will get another hash value because of different creation time and author data.
```

Comment:
Instead of "another", can you use "a different"? I initially read this as "you'll get a second hash value."